docs/developer/manifests/values.dev.yaml

```diff
@@ -0,0 +1,197 @@
+  ## @param kubeappsapis.podSecurityContext.enabled Enabled KubeappsAPIs pods' Security Context
+  ## @param kubeappsapis.podSecurityContext.fsGroup Set KubeappsAPIs pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context (only main container)
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ## @param kubeappsapis.containerSecurityContext.enabled Enabled KubeappsAPIs containers' Security Context
+  ## @param kubeappsapis.containerSecurityContext.runAsUser Set KubeappsAPIs container's Security Context runAsUser
+  ## @param kubeappsapis.containerSecurityContext.runAsNonRoot Set KubeappsAPIs container's Security Context runAsNonRoot
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 1001
+    runAsNonRoot: true
+  ## Configure extra options for KubeappsAPIs containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param kubeappsapis.livenessProbe.enabled Enable livenessProbe
+  ## @skip kubeappsapis.livenessProbe.httpGet
+  ## @param kubeappsapis.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param kubeappsapis.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param kubeappsapis.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param kubeappsapis.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param kubeappsapis.livenessProbe.successThreshold Success threshold for livenessProbe
+  ## KubeappsAPIs containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ##
+  livenessProbe:
+    enabled: false
+    httpGet:
+      path: /live
+      port: 50051
+    initialDelaySeconds: 60
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  ## @param kubeappsapis.readinessProbe.enabled Enable readinessProbe
+  ## @skip kubeappsapis.readinessProbe.httpGet
+  ## @param kubeappsapis.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param kubeappsapis.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param kubeappsapis.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param kubeappsapis.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param kubeappsapis.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: false
+    httpGet:
+      path: /ready
+      port: 50051
+    initialDelaySeconds: 0
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
```

Comment:
We are not defining a `/ready` nor a `/live, aren't we?

Reply:
Nope, which is why I've set both `readinessProbe.enabled` and `livenessProbe.enabled` to false for now.

Reply:
Hah - :facepalm: I didn't notice the `enabled: false`